github/install.go

```diff
@@ -0,0 +1,232 @@
+package github
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	"github.com/dlclark/regexp2"
+	"github.com/packwiz/packwiz/core"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var GithubRegex = regexp.MustCompile(`^https?://(?:www\.)?github\.com/([^/]+/[^/]+)`)
+
+// installCmd represents the install command
+var installCmd = &cobra.Command{
+	Use:     "add [URL|slug]",
+	Short:   "Add a project from a GitHub repository URL or slug",
```

Comment:
Could you add some examples of valid URLs / slugs to use? Slug isn't a super common term and probably confuses people (yes the rest of packwiz is guilty of this).

Reply:
Do you want it in the command description, or in the online documentation?

Reply:
The online reference docs are generated from the cobra commands, so if there's documentation in there it'll show up in both places!